lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1581,6 +1592,11 @@ public function get_collection_params() {
 
 		$params['context']['default'] = 'view';
 
+		$params['after'] = array(
+			'description'       => __( 'The after parameter in a date query, send as an ISO8601 or strtotime() compliant date string. See wp-includes/date.php' ),
```

Comment:
Accepting `strtotime()`-compatible values seems a bit sketchy to me, given we don't accept them anywhere else in the API and they're very much a PHP-ism.

If we change `format=>date-time`, then we get validation for ISO8601 for free and don't have to introduce a validation callback.